src/components/application_manager/test/commands/mobile/simple_notification_commands_test.cc

```diff
@@ -68,6 +69,7 @@ typedef Types<commands::OnAppInterfaceUnregisteredNotification,
               commands::OnAudioPassThruNotification,
               commands::OnLanguageChangeNotification,
               commands::OnPermissionsChangeNotification,
+              commands::mobile::OnSeekMediaClockTimerNotification,
```

Comment:
@indlu I think that changes in this file should be placed in commit related to unit tests.